README.md

```diff
@@ -1,6 +1,25 @@
-# oncotree
+# OncoTree
+
+## General
+
+The OncoTree was developed at [Memorial Sloan Kettering Cancer Center](https://www.mskcc.org/) (MSK) as a resource for standardizing detailed cancer type information and mapping these details to unique accessions called OncoTree codes.
```

Comment:
I would emphasize the open-source aspect of the project. We would like the widest possible participation and use of oncotree. Identifying it as an MSK resource might discourage adoption maybe?

Reply:
Maybe mention that it is open source but keep that it was developed at MSK?

Reply:
@sheridancbio I think we share elsewhere that OncoTree was developed at MSK, or maybe I'm thinking of OncoKB? Either way I think it's fine that we include where it was developed. I'll add that's open-source though, thanks for that suggestion